tools/packaging/common/envoy_bootstrap_v2.json

```diff
@@ -333,7 +333,10 @@
               ],
               "validation_context": {
                 "trusted_ca": {
-                  {{- if eq .pilot_cert_provider "kubernetes" }}
+                  {{- if .provided_cert }}
```

Comment:
Is .provided_cert only used for the VM case?  Can you add a description of the intended use case or workflow for .provded_cert to the PR description/code?

Reply:
added description in this PR to describe the flag usage. Please take another look. Thanks.

Reply:
Thanks! Approved.